test_unstructured/partition/xlsx/test_xlsx.py

```diff
@@ -32,17 +41,17 @@ def test_partition_xlsx_from_filename(filename="example-docs/stanley-cups.xlsx")
     assert elements[1].metadata.filename == "stanley-cups.xlsx"
 
 
-def test_partition_xlsx_from_filename_with_emoji(filename="example-docs/emoji.xlsx"):
-    elements = partition_xlsx(filename=filename, include_header=False)
+def test_partition_xlsx_from_filename_with_emoji():
+    elements = partition_xlsx("example-docs/emoji.xlsx", include_header=False)
```

Comment:
Several of the test-cleanup changes are removing the `filename="some-document-path"` parameter from the test function and inlining `"some-document-path"` in the partition call.

The only caller of a test function is `pytest` (the test-runner) and it will never provide a `filename` argument. This simplification makes the test more direct-reading and reduces noise.